### ISSUE TITLE
UI Issue with self-hosted Supabase

### ISSUE THREAD

**Issue by M1ck0** [supabase/supabase#28531]
# Bug report

- [x] I confirm this is a bug with Supabase, not with my own application. 
- [x] I confirm I have searched the [Docs](https://docs.supabase.com), GitHub [Discussions](https://github.com/supabase/supabase/discussions), and [Discord](https://discord.supabase.com). 

## Describe the bug

There is a UI bug in self-hosted version of Supabase

## To Reproduce

There are no errors in the browser console, so I can't debug the issue myself. All CSS files are loaded and I don't have any extension that would mess with this. It also happens in Incognito with all extensions disabled

## Screenshots
<img width="1379" alt="Screenshot 2024-08-11 at 10 59 38" src="https://github.com/user-attachments/assets/a78aeb91-e336-49d3-b8fd-c8159f27dd85">

## System information

- OS: Supabase is running on Linux
- Browser (if applies): All of them
- Version of supabase-js: Latest

**Comment by Hallidayo** [comment]
Hi @M1ck0 - thank you for opening the issue. Do you have steps to recreate? Tried locally but I am unable to at the moment.

**Comment by saltcod** [comment]
Are you running the latest version @M1ck0 ?

**Comment by M1ck0** [comment]
@Hallidayo I do not know how to recreate it. I installed Supabase about a week ago and everything was fine, but 2 days ago this bug happened.

It is installed through Coolify on my private VPS

**Comment by M1ck0** [comment]
@saltcod Yes, yes I do. It is managed by Coolify on my private VPS, and the issue just happened randomly 2 days ago.

**Comment by M1ck0** [comment]
[This](https://github.com/supabase/supabase/issues/28481#issuecomment-2285729203) comments fixes the issue for now

**Comment by Hallidayo** [comment]
Hi all, We’re going to close this one due to inactivity. Please feel free to reopen if you’re still having issues here or open a new issue.
